examples/input/keyboard_input.rs

```diff
@@ -1,9 +1,6 @@
 //! Demonstrates handling a key press/release.
 
-use bevy::{
-    input::{keyboard::KeyCode, Input},
-    prelude::*,
-};
+use bevy::prelude::*;
```

Comment:
What's the purpose of this import refactor?
I understand the point that `bevy::prelude::*` brings in `input::*` so we don't _techinally_ need to use the `input::` import directly.

However, because this is an example - a new user who is viewing this might like to how where and how the `Input` or `KeyCode` types come from.
I'm personally a fan of the redundant imports.

(ditto on the rest of the similar changes in the MR)

Reply:
I like removing duplicate imports like this in examples in favor of prelude. In engine code itself, I'd prefer to move away from prelude imports though.

Reply:
I agree with Alice.

Examples should be minimal.
> a new user who is viewing this might like to how where and how the Input or KeyCode types come from.

The vast majority of users don't care where a type comes from within Bevy and will rely on the prelude and auto-importing.
If a user does want to know they should rely on their IDE or the docs.

Reply:
sounds good to me